### DISCUSSION TITLE
Adding command prefix ignores Steam Play

### DISCUSSION THREAD

**Discussion by jonpas** [lutris/lutris#3373] [Q&A]
Steam for Linux with a manually-enabled Steam Play game (not a whitelisted one, but probably same story). Adding "Command prefix" (such as `primusrun`) makes Steam ignore Proton completely and tries to start it natively (it seems) and of course fails.

It also makes Steam's launch parameters be ignored, while they are used just fine if "Command prefix" is left empty. Moving launch parameters from Steam to Lutris doesn't help (results in Proton not being used).

Not giving any command prefix parameters works fine, but that requires all configuration to be done in Steam.

---

Unrelated but thought I'd mention it here as I am not sure if it was noted before. I can move it to a separate issue to track it nicer if you wish.

It would be great to have a Proton selection in Lutris itself, like we do in Steam, if that's doable. That would allow running games with eg. `primusrun` without running entire Steam under it (like with "Optimus launcher" option).

Additionally, it would open some other possibilities, like running Wine-supported games with a lot of dependencies to run through Steam Play (eg. Assetto Corsa with Wine and winetricks -> Assetto Corsa with Proton and protontricks).

And of course provide one program to configure games from, instead of back and forth between Steam and Lutris.

**Comment by strycore** [comment]
I'll have a look at the command prefix issue, looks very weird.

For configuring Proton games from Lutris itself, I think it's covered in another ticket so it should be doable.

**Comment by jonpas** [comment]
Cheers! Note, I was testing mostly with Ori and the Blind Forest: Definitive Edition, since it's not whitelisted and uses Vulkan. Using primus vk due to GPU pass-through.

**Comment by strycore** [comment]
Moving to Discussions since I do not have an answer for this, I cannot test anything Optimus related.
